website/static/building-stylus-faqs.json

```diff
@@ -2,5 +2,10 @@
 {"question": "Is there any analogue of the fallback function from Solidity in the Rust Stylus SDK?","answer": "<p>Currently there isn't any analogue. However, you can use a minimal entrypoint and perform raw delegate calls, forwarding your calldata. You can find more information in <a href=\"https://docs.arbitrum.io/stylus/reference/rust-sdk-guide#bytes-in-bytes-out-programming\">Bytes-in, bytes-out programming</a> and <a href=\"https://docs.arbitrum.io/stylus/reference/rust-sdk-guide#call-static_call-and-delegate_call\">call, static_call and delegate_call</a>.</p>\n\n<p></p>\n\n","key": "is-there-any-analogue-of-the-fallback-function-from-solidity-in-the-rust-stylus-sdk"},
 {"question": "Why are constructors not yet supported for Stylus contracts?","answer": "<p>Constructors use EVM bytecode to initialize state. While one could add EVM bytecode manually to their Stylus deployment, we don't allow WASM execution in the constructor so there's no way to express this in the SDK.</p>\n\n<p>We're working on models that will make init easier, so there might be better solutions available in the future. For now, we suggest calling an init method after deploying.</p>\n\n<p></p>\n\n","key": "why-are-constructors-not-yet-supported-for-stylus-contracts"},
-{"question": "Is it possible to verify Stylus contracts on the block explorer?","answer": "<p>Currently it is not possible to verify contracts compiled to WASM on the block explorer, but we are actively working with providers to have the verification process ready for when Stylus reaches mainnet-ready status.</p>\n\n<p></p>\n\n","key": "is-it-possible-to-verify-stylus-contracts-on-the-block-explorer"}
+{"question": "Is it possible to verify Stylus contracts on the block explorer?","answer": "<p>Currently it is not possible to verify contracts compiled to WASM on the block explorer, but we are actively working with providers to have the verification process ready for when Stylus reaches mainnet-ready status.</p>\n\n<p></p>\n\n","key": "is-it-possible-to-verify-stylus-contracts-on-the-block-explorer"},
+{"question": "Do Stylus contracts compile down to EVM bytecode like prior other attempts?","answer": "<p>No. Stylus contracts are compiled down to WASM. The user writes a program in Rust / C / C++ which is then compiled down to WebAssembly.</p>\n\n<p></p>\n\n","key": "do-stylus-contracts-compile-down-to-evm-bytecode-like-prior-other-attempts"},
+{"question": "How is a Stylus contract deployed?","answer": "<p>Stylus contracts are deployed on chain as a blob of bytes, just like EVM ones. The only difference is that when the contract is executed, instead of invoking the EVM, we invoke a separate WASM runtime. The way you tell apart the two is by a special EOF-inspired prefix. When a contract code starts with the magic <code>0xEF000000</code> prefix, it's a WASM. </p>\n\n<p></p>\n\n","key": "how-is-a-stylus-contract-deployed"},
```

Comment:
```suggestion
{"question": "How is a Stylus contract deployed?","answer": "<p>Stylus contracts are deployed on chain as a blob of bytes, just like EVM ones. The only difference is that when the contract is executed, instead of invoking the EVM, we invoke a separate WASM runtime. Note that a special EOF-inspired prefix distinguishes Stylus contracts from traditional EVM contracts: when a contract's bytecode starts with the magic <code>0xEF000000</code> prefix, it's a Stylus WASM. </p>\n\n<p></p>\n\n","key": "how-is-a-stylus-contract-deployed"},
```

nits